src/services/auth/auth.service.ts

```diff
@@ -41,37 +41,38 @@ export class AuthService {
 
   async login(user: any) {
     console.log(user);
-    const payload = { username: user.email, sub: user._id };
+    const payload: User = { username: user.email, sub: user._id, network: 'email' };
```

Comment:
This implies that `sub` is a mongodb ObjectId, but below in `generateSub`, `sub` is not a mongodb ObjectId. What's with that?